gfortran 11.4 cannot compile - /usr/bin/ld: cannot find -lFALSE: No such file or directory

The only other issue I found like this is https://github.com/dftd4/dftd4/issues/124, which was blamed on CMake 3.17 or Cray.  I am using GCC 11.4 on an Ubuntu 22 system, so I don't understand the problem.

What should I do to debug this?

```
/tmp$ git clone --recursive https://github.com/dftd4/dftd4.git && cd dftd4 && mkdir build && cd build && cmake .. && cmake --build .
Cloning into 'dftd4'...
remote: Enumerating objects: 2141, done.
remote: Counting objects: 100% (710/710), done.
remote: Compressing objects: 100% (448/448), done.
remote: Total 2141 (delta 558), reused 345 (delta 259), pack-reused 1431 (from 1)
Receiving objects: 100% (2141/2141), 2.09 MiB | 10.82 MiB/s, done.
Resolving deltas: 100% (1459/1459), done.
Submodule 'assets/aur/dftd4' (https://aur.archlinux.org/dftd4.git) registered for path 'assets/aur/dftd4'
Submodule 'assets/aur/dftd4-git' (https://aur.archlinux.org/dftd4-git.git) registered for path 'assets/aur/dftd4-git'
Cloning into '/tmp/dftd4/assets/aur/dftd4'...
remote: Enumerating objects: 26, done.
remote: Counting objects: 100% (26/26), done.
remote: Compressing objects: 100% (24/24), done.
remote: Total 26 (delta 2), reused 26 (delta 2), pack-reused 0 (from 0)
Receiving objects: 100% (26/26), 11.62 KiB | 11.62 MiB/s, done.
Resolving deltas: 100% (2/2), done.
Cloning into '/tmp/dftd4/assets/aur/dftd4-git'...
remote: Enumerating objects: 8, done.
remote: Counting objects: 100% (8/8), done.
remote: Compressing objects: 100% (8/8), done.
remote: Total 8 (delta 0), reused 8 (delta 0), pack-reused 0 (from 0)
Receiving objects: 100% (8/8), done.
Submodule path 'assets/aur/dftd4': checked out 'a8947fb60f1b6d192046cb91c7c0524fd4470885'
Submodule path 'assets/aur/dftd4-git': checked out '7e92bd04dea45dcba5d127938d55e85f73c6e753'
-- The Fortran compiler identification is GNU 11.4.0
-- Detecting Fortran compiler ABI info
-- Detecting Fortran compiler ABI info - done
-- Check for working Fortran compiler: /usr/bin/f95 - skipped
-- Setting build type to 'RelWithDebInfo' as none was specified.
-- Found OpenMP_Fortran: -fopenmp (found version "4.5")
-- Found OpenMP: TRUE (found version "4.5")
-- Looking for Fortran sgemm
-- Looking for Fortran sgemm - not found
-- Looking for Fortran sgemm
-- Looking for Fortran sgemm - found
-- Found BLAS: /usr/lib/x86_64-linux-gnu/libblas.so
-- mctc-lib: Find installed package
-- Could NOT find mctc-lib (missing: mctc-lib_DIR)
-- Retrieving mctc-lib revision v0.3.0 from https://github.com/grimme-lab/mctc-lib
-- mstore: Find installed package
-- Could NOT find mstore (missing: mstore_DIR)
-- Retrieving mstore revision v0.2.0 from https://github.com/grimme-lab/mstore
-- multicharge: Find installed package
-- Could NOT find multicharge (missing: multicharge_DIR)
-- Retrieving multicharge revision HEAD from https://github.com/grimme-lab/multicharge
-- Could NOT find LAPACK (missing: LAPACK_LIBRARIES)
-- The C compiler identification is GNU 11.4.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Configuring done (3.6s)
-- Generating done (0.0s)
-- Build files have been written to: /tmp/dftd4/build
[  0%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/env/accuracy.f90.o
[  1%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/env/error.f90.o
[  2%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/env/system.f90.o
[  3%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/env.f90.o
[  3%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/env/testing.f90.o
[  4%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/filetype.f90.o
[  5%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/codata2018.f90.o
[  6%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/constants.f90.o
[  6%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/convert.f90.o
[  7%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/resize.f90.o
[  8%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/structure/info.f90.o
[  9%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/symbols.f90.o
[ 10%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/structure.f90.o
[ 10%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/utils.f90.o
[ 11%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/read/aims.f90.o
[ 12%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/read/ctfile.f90.o
[ 12%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/read/gaussian.f90.o
[ 13%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/read/genformat.f90.o
[ 14%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/read/pdb.f90.o
[ 15%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/read/qchem.f90.o
[ 15%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/read/qcschema.F90.o
[ 16%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/read/turbomole.f90.o
[ 16%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/read/vasp.f90.o
[ 17%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/read/xyz.f90.o
[ 17%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/read.f90.o
[ 17%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/write/aims.f90.o
[ 18%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/write/ctfile.f90.o
[ 19%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/write/gaussian.f90.o
[ 20%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/math.f90.o
[ 20%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/write/genformat.f90.o
[ 21%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/write/pdb.f90.o
[ 22%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/write/qchem.f90.o
[ 23%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/version.F90.o
[ 23%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/write/qcschema.f90.o
[ 24%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/write/turbomole.f90.o
[ 25%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/write/vasp.f90.o
[ 25%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/write/xyz.f90.o
[ 26%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io/write.f90.o
[ 26%] Building Fortran object _deps/mctc-lib-build/CMakeFiles/mctc-lib-lib.dir/src/mctc/io.f90.o
[ 27%] Linking Fortran static library libmctc-lib.a
[ 27%] Built target mctc-lib-lib
[ 28%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge/cutoff.f90.o
[ 29%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge/data/covrad.f90.o
[ 30%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge/data.f90.o
[ 30%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge/blas.F90.o
[ 30%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge/ewald.f90.o
[ 31%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge/lapack.F90.o
[ 32%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge/wignerseitz.f90.o
[ 33%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge/model.F90.o
[ 33%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge/ncoord.f90.o
[ 33%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge/version.f90.o
[ 34%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge/output.f90.o
[ 35%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge/param/eeq2019.f90.o
[ 36%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge/param.f90.o
[ 37%] Building Fortran object _deps/multicharge-build/CMakeFiles/multicharge-lib.dir/src/multicharge.f90.o
[ 37%] Linking Fortran static library libmulticharge.a
[ 37%] Built target multicharge-lib
[ 38%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/cutoff.f90.o
[ 38%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/damping.f90.o
[ 39%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/damping/atm.f90.o
[ 40%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/data/covrad.f90.o
[ 41%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/data/en.f90.o
[ 41%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/data/hardness.f90.o
[ 42%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/data/r4r2.f90.o
[ 43%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/data/zeff.f90.o
[ 44%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/data.f90.o
[ 44%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/damping/rational.f90.o
[ 44%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/blas.F90.o
[ 45%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/charge.f90.o
[ 46%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/reference.f90.o
[ 46%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/model.f90.o
[ 47%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/ncoord.f90.o
[ 48%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/disp.f90.o
[ 49%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/numdiff.f90.o
[ 49%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/utils.f90.o
[ 50%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/param.f90.o
[ 51%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/version.f90.o
[ 51%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4.f90.o
[ 52%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/api.f90.o
[ 52%] Building Fortran object CMakeFiles/dftd4-lib.dir/src/dftd4/output.f90.o
[ 53%] Linking Fortran static library libdftd4.a
[ 53%] Built target dftd4-lib
[ 54%] Building Fortran object _deps/mctc-lib-build/app/CMakeFiles/mctc-convert.dir/main.f90.o
[ 55%] Linking Fortran executable mctc-convert
[ 55%] Built target mctc-convert
[ 56%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_math.f90.o
[ 56%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_read.f90.o
[ 57%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_read_aims.f90.o
[ 58%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_read_ctfile.f90.o
[ 58%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_read_gaussian.f90.o
[ 59%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_read_genformat.f90.o
[ 60%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_read_pdb.f90.o
[ 60%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_read_qchem.f90.o
[ 61%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_read_qcschema.f90.o
[ 62%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_read_turbomole.f90.o
[ 62%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_read_vasp.f90.o
[ 63%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_read_xyz.f90.o
[ 64%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_symbols.f90.o
[ 65%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/testsuite_structure.f90.o
[ 65%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_write.f90.o
[ 66%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_write_aims.f90.o
[ 67%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_write_ctfile.f90.o
[ 67%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_write_gaussian.f90.o
[ 68%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_write_genformat.f90.o
[ 69%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_write_pdb.f90.o
[ 69%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_write_qchem.f90.o
[ 70%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_write_turbomole.f90.o
[ 71%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_write_vasp.f90.o
[ 71%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/test_write_xyz.f90.o
[ 71%] Building Fortran object _deps/mctc-lib-build/test/CMakeFiles/mctc-lib-tester.dir/main.f90.o
[ 72%] Linking Fortran executable mctc-lib-tester
[ 72%] Built target mctc-lib-tester
[ 73%] Building Fortran object _deps/mstore-build/CMakeFiles/mstore-lib.dir/src/mstore/data/record.f90.o
[ 73%] Building Fortran object _deps/mstore-build/CMakeFiles/mstore-lib.dir/src/mstore/amino20x4.f90.o
[ 74%] Building Fortran object _deps/mstore-build/CMakeFiles/mstore-lib.dir/src/mstore/but14diol.f90.o
[ 74%] Building Fortran object _deps/mstore-build/CMakeFiles/mstore-lib.dir/src/mstore/data/collection.f90.o
[ 75%] Building Fortran object _deps/mstore-build/CMakeFiles/mstore-lib.dir/src/mstore/data/store.f90.o
[ 76%] Building Fortran object _deps/mstore-build/CMakeFiles/mstore-lib.dir/src/mstore/heavy28.f90.o
[ 76%] Building Fortran object _deps/mstore-build/CMakeFiles/mstore-lib.dir/src/mstore/ice10.f90.o
[ 77%] Building Fortran object _deps/mstore-build/CMakeFiles/mstore-lib.dir/src/mstore/il16.f90.o
[ 78%] Building Fortran object _deps/mstore-build/CMakeFiles/mstore-lib.dir/src/mstore/mb16_43.f90.o
[ 78%] Building Fortran object _deps/mstore-build/CMakeFiles/mstore-lib.dir/src/mstore/upu23.f90.o
[ 79%] Building Fortran object _deps/mstore-build/CMakeFiles/mstore-lib.dir/src/mstore/x23.f90.o
[ 80%] Building Fortran object _deps/mstore-build/CMakeFiles/mstore-lib.dir/src/mstore.f90.o
[ 80%] Building Fortran object _deps/mstore-build/CMakeFiles/mstore-lib.dir/src/mstore_version.f90.o
[ 81%] Linking Fortran static library libmstore.a
[ 81%] Built target mstore-lib
[ 82%] Building Fortran object _deps/mstore-build/app/fortranize/CMakeFiles/mstore-fortranize.dir/main.f90.o
[ 82%] Linking Fortran executable mstore-fortranize
[ 82%] Built target mstore-fortranize
[ 83%] Building Fortran object _deps/mstore-build/app/info/CMakeFiles/mstore-info.dir/main.f90.o
[ 84%] Linking Fortran executable mstore-info
[ 84%] Built target mstore-info
[ 85%] Building Fortran object _deps/multicharge-build/app/CMakeFiles/multicharge-exe.dir/main.f90.o
[ 85%] Linking Fortran executable multicharge
/usr/bin/ld: cannot find -lFALSE: No such file or directory
collect2: error: ld returned 1 exit status
gmake[2]: *** [_deps/multicharge-build/app/CMakeFiles/multicharge-exe.dir/build.make:101: _deps/multicharge-build/app/multicharge] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:921: _deps/multicharge-build/app/CMakeFiles/multicharge-exe.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2
```

```
$ gfortran -v
Using built-in specs.
COLLECT_GCC=gfortran
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/11/lto-wrapper
OFFLOAD_TARGET_NAMES=nvptx-none:amdgcn-amdhsa
OFFLOAD_TARGET_DEFAULT=1
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu 11.4.0-1ubuntu1~22.04' --with-bugurl=file:///usr/share/doc/gcc-11/README.Bugs --enable-languages=c,ada,c++,go,brig,d,fortran,objc,obj-c++,m2 --prefix=/usr --with-gcc-major-version-only --program-suffix=-11 --program-prefix=x86_64-linux-gnu- --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --libdir=/usr/lib --enable-nls --enable-bootstrap --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --with-default-libstdcxx-abi=new --enable-gnu-unique-object --disable-vtable-verify --enable-plugin --enable-default-pie --with-system-zlib --enable-libphobos-checking=release --with-target-system-zlib=auto --enable-objc-gc=auto --enable-multiarch --disable-werror --enable-cet --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --enable-multilib --with-tune=generic --enable-offload-targets=nvptx-none=/build/gcc-11-XeT9lY/gcc-11-11.4.0/debian/tmp-nvptx/usr,amdgcn-amdhsa=/build/gcc-11-XeT9lY/gcc-11-11.4.0/debian/tmp-gcn/usr --without-cuda-driver --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu --with-build-config=bootstrap-lto-lean --enable-link-serialization=2
Thread model: posix
Supported LTO compression algorithms: zlib zstd
gcc version 11.4.0 (Ubuntu 11.4.0-1ubuntu1~22.04)
```

```
$ cmake --version
cmake version 3.29.1

CMake suite maintained and supported by Kitware (kitware.com/cmake).
```

Removing the `-lFALSE` suggests that CMake has done something stupid with LAPACK detection:

```
/tmp/dftd4/build/_deps/multicharge-build/app$ /usr/bin/f95 -O2 -g "CMakeFiles/multicharge-exe.dir/main.f90.o" -o multicharge  ../libmulticharge.a ../../mctc-lib-build/libmctc-lib.a /usr/lib/x86_64-linux-gnu/libblas.so /usr/lib/gcc/x86_64-linux-gnu/11/libgomp.so /usr/lib/x86_64-linux-gnu/libpthread.a
/usr/bin/ld: ../libmulticharge.a(lapack.F90.o): in function `__multicharge_lapack_MOD_mchrg_dsytri':
/tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:353: undefined reference to `dsytri_'
/usr/bin/ld: /tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:353: undefined reference to `dsytri_'
/usr/bin/ld: /tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:353: undefined reference to `dsytri_'
/usr/bin/ld: ../libmulticharge.a(lapack.F90.o): in function `__multicharge_lapack_MOD_mchrg_ssytri':
/tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:322: undefined reference to `ssytri_'
/usr/bin/ld: /tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:322: undefined reference to `ssytri_'
/usr/bin/ld: /tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:322: undefined reference to `ssytri_'
/usr/bin/ld: ../libmulticharge.a(lapack.F90.o): in function `__multicharge_lapack_MOD_mchrg_dsytrs':
/tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:247: undefined reference to `dsytrs_'
/usr/bin/ld: ../libmulticharge.a(lapack.F90.o): in function `__multicharge_lapack_MOD_mchrg_ssytrs':
/tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:221: undefined reference to `ssytrs_'
/usr/bin/ld: ../libmulticharge.a(lapack.F90.o): in function `__multicharge_lapack_MOD_mchrg_dsytrf':
/tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:183: undefined reference to `dsytrf_'
/usr/bin/ld: /tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:190: undefined reference to `dsytrf_'
/usr/bin/ld: /tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:183: undefined reference to `dsytrf_'
/usr/bin/ld: /tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:190: undefined reference to `dsytrf_'
/usr/bin/ld: /tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:183: undefined reference to `dsytrf_'
/usr/bin/ld: ../libmulticharge.a(lapack.F90.o):/tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:190: more undefined references to `dsytrf_' follow
/usr/bin/ld: ../libmulticharge.a(lapack.F90.o): in function `__multicharge_lapack_MOD_mchrg_ssytrf':
/tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:144: undefined reference to `ssytrf_'
/usr/bin/ld: /tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:151: undefined reference to `ssytrf_'
/usr/bin/ld: /tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:144: undefined reference to `ssytrf_'
/usr/bin/ld: /tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:151: undefined reference to `ssytrf_'
/usr/bin/ld: /tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:144: undefined reference to `ssytrf_'
/usr/bin/ld: ../libmulticharge.a(lapack.F90.o):/tmp/dftd4/build/_deps/multicharge-src/src/multicharge/lapack.F90:151: more undefined references to `ssytrf_' follow
collect2: error: ld returned 1 exit status
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -19,7 +19,7 @@ cmake_minimum_required(VERSION 3.14)
 project(
   "dftd4"
   LANGUAGES "Fortran"
-  VERSION "3.6.0"
+  VERSION "3.7.0"
   DESCRIPTION "Generally Applicable Atomic-Charge Dependent London Dispersion Correction"
 )
 

--- a/README.md
+++ b/README.md
@@ -294,13 +294,13 @@ pkg-config --modversion dftd4
 
 If your ``dftd4`` installation is not findable, you have to update your environment variables.
 One option is to provide a module file for your ``dftd4`` installation.
-The example module file below can be placed in your ``MODULEPATH`` to provide access to an installation in ``~/opt/dftd4/3.6.0``.
+The example module file below can be placed in your ``MODULEPATH`` to provide access to an installation in ``~/opt/dftd4/3.7.0``.
 Retry the above comment after loading the ``dftd4`` module and adjust the module file until ``pkg-config`` finds your installation.
 
 ```lua
--- dftd4/3.6.0.lua
+-- dftd4/3.7.0.lua
 local name = "dftd4"
-local version = "3.6.0"
+local version = "3.7.0"
 local prefix = pathJoin(os.getenv("HOME"), "opt", name, version)
 local libdir = "lib"  -- or lib64
 

--- a/config/cmake/Findcustom-blas.cmake
+++ b/config/cmake/Findcustom-blas.cmake
@@ -20,7 +20,7 @@ if(NOT BLAS_FOUND)
     # https://cmake.org/cmake/help/v3.14/module/FindBLAS.html#result-variables
     enable_language("C")
   endif()
-  find_package("BLAS")
+  find_package("BLAS" REQUIRED)
   if(NOT TARGET "BLAS::BLAS")
     add_library("BLAS::BLAS" INTERFACE IMPORTED)
     target_link_libraries("BLAS::BLAS" INTERFACE "${BLAS_LIBRARIES}")

--- a/config/cmake/Findmctc-lib.cmake
+++ b/config/cmake/Findmctc-lib.cmake
@@ -17,7 +17,7 @@
 set(_lib "mctc-lib")
 set(_pkg "MCTCLIB")
 set(_url "https://github.com/grimme-lab/mctc-lib")
-set(_revision "v0.3.0")
+set(_revision "v0.3.2")
 
 if(NOT DEFINED "${_pkg}_FIND_METHOD")
   if(DEFINED "${PROJECT_NAME}-dependency-method")

--- a/config/cmake/Findmstore.cmake
+++ b/config/cmake/Findmstore.cmake
@@ -17,7 +17,7 @@
 set(_lib "mstore")
 set(_pkg "MSTORE")
 set(_url "https://github.com/grimme-lab/mstore")
-set(_revision "v0.2.0")
+set(_revision "v0.3.0")
 
 if(NOT DEFINED "${_pkg}_FIND_METHOD")
   if(DEFINED "${PROJECT_NAME}-dependency-method")

--- a/config/cmake/Findmulticharge.cmake
+++ b/config/cmake/Findmulticharge.cmake
@@ -17,7 +17,7 @@
 set(_lib "multicharge")
 set(_pkg "MULTICHARGE")
 set(_url "https://github.com/grimme-lab/multicharge")
-set(_revision "HEAD")
+set(_revision "v0.3.0")
 
 if(NOT DEFINED "${_pkg}_FIND_METHOD")
   if(DEFINED "${PROJECT_NAME}-dependency-method")

--- a/doc/conf.py
+++ b/doc/conf.py
@@ -8,7 +8,7 @@ import dftd4
 
 project = "DFT-D4"
 author = "Grimme group"
-copyright = f"2017–2023 {author}"
+copyright = f"2017–2024 {author}"
 
 extensions = [
     "sphinx_copybutton",

--- a/doc/recipe/vasp.rst
+++ b/doc/recipe/vasp.rst
@@ -18,14 +18,14 @@ After you completed the installation of ``dftd4``, make sure it is findable by `
 
 If your ``dftd4`` installation is not findable, you have to update your environment variables.
 One option is to provide a module file for your ``dftd4`` installation.
-The example module file below can be placed in your ``MODULEPATH`` to provide access to an installation in ``~/opt/dftd4/3.6.0``.
+The example module file below can be placed in your ``MODULEPATH`` to provide access to an installation in ``~/opt/dftd4/3.7.0``.
 Retry the above comment after loading the ``dftd4`` module and adjust the module file until ``pkg-config`` finds your installation.
 
 .. code-block:: lua
 
-   -- dftd4/3.6.0.lua
+   -- dftd4/3.7.0.lua
    local name = "dftd4"
-   local version = "3.6.0"
+   local version = "3.7.0"
    local prefix = pathJoin(os.getenv("HOME"), "opt", name, version)
    local libdir = "lib"  -- or lib64
 

--- a/fpm.toml
+++ b/fpm.toml
@@ -1,5 +1,5 @@
 name = "dftd4"
-version = "3.6.0"
+version = "3.7.0"
 license = "LGPL-3.0-or-later"
 maintainer = ["@awvwgk"]
 author = ["Eike Caldeweyher", "Sebastian Ehlert", "Stefan Grimme"]

--- a/meson.build
+++ b/meson.build
@@ -17,7 +17,7 @@
 project(
   'dftd4',
   'fortran',
-  version: '3.6.0',
+  version: '3.7.0',
   license: 'LGPL-3.0-or-later',
   meson_version: '>=0.55',
   default_options: [

--- a/python/README.rst
+++ b/python/README.rst
@@ -237,7 +237,7 @@ Now you are ready to use ``dftd4``, check if you can import it with
    >>> import dftd4
    >>> from dftd4.libdftd4 import get_api_version
    >>> get_api_version()
-   '3.6.0'
+   '3.7.0'
 
 
 Building the extension module

--- a/python/dftd4/__init__.py
+++ b/python/dftd4/__init__.py
@@ -18,4 +18,4 @@
 # make sure we have a CFFI available
 import cffi
 
-__version__ = "3.6.0"
+__version__ = "3.7.0"

--- a/python/dftd4/library.py
+++ b/python/dftd4/library.py
@@ -25,7 +25,7 @@ Example
 -------
 >>> from dftd4.library import get_api_version
 >>> get_api_version()
-'3.6.0'
+'3.7.0'
 """
 
 import functools

--- a/python/meson.build
+++ b/python/meson.build
@@ -19,7 +19,7 @@
 project(
   'dftd4',
   'c',
-  version: '3.6.0',
+  version: '3.7.0',
   license: 'LGPL-3.0-or-later',
   meson_version: '>=0.55',
   default_options: [

--- a/python/pyproject.toml
+++ b/python/pyproject.toml
@@ -4,7 +4,7 @@ build-backend = "mesonpy"
 
 [project]
 name = "dftd4"
-version = "3.6.0"
+version = "3.7.0"
 description = "Python API of the DFT-D4 project"
 readme = "README.rst"
 license.text = "LGPL-3.0-or-later"
@@ -18,6 +18,7 @@ classifiers = [
   "Programming Language :: Python :: 3.9",
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
   "Topic :: Scientific/Engineering :: Chemistry",
   "Topic :: Scientific/Engineering :: Physics",
 ]

--- a/src/dftd4/version.f90
+++ b/src/dftd4/version.f90
@@ -24,10 +24,10 @@ module dftd4_version
 
 
    !> String representation of the dftd4 version
-   character(len=*), parameter :: dftd4_version_string = "3.6.0"
+   character(len=*), parameter :: dftd4_version_string = "3.7.0"
 
    !> Numeric representation of the dftd4 version
-   integer, parameter :: dftd4_version_compact(3) = [3, 6, 0]
+   integer, parameter :: dftd4_version_compact(3) = [3, 7, 0]
 
 
 contains

--- a/subprojects/mctc-lib.wrap
+++ b/subprojects/mctc-lib.wrap
@@ -1,4 +1,4 @@
 [wrap-git]
 directory = mctc-lib
 url = https://github.com/grimme-lab/mctc-lib
-revision = v0.3.0
+revision = v0.3.2

--- a/subprojects/mstore.wrap
+++ b/subprojects/mstore.wrap
@@ -1,4 +1,4 @@
 [wrap-git]
 directory = mstore
 url = https://github.com/grimme-lab/mstore
-revision = v0.2.0
+revision = v0.3.0

--- a/subprojects/multicharge.wrap
+++ b/subprojects/multicharge.wrap
@@ -1,4 +1,4 @@
 [wrap-git]
 directory = multicharge
 url = https://github.com/grimme-lab/multicharge
-revision = head
+revision = v0.3.0

